src/ranged.cpp

```diff
@@ -1536,14 +1536,34 @@ void drop_or_embed_projectile( const dealt_projectile_attack &attack )
     item dropped_item = drop_item;
 
     monster *mon = dynamic_cast<monster *>( attack.hit_critter );
-    // Try to embed the projectile in monster
-    // Don't embed on miss, in player/NPCs, when we didn't stab/cut properly
-    //  or when the item simply shouldn't be embedded (for example, it is active)
-    if( mon == nullptr || mon->is_dead_state() ||
-        ( attack.dealt_dam.type_damage( DT_STAB ) +
-          attack.dealt_dam.type_damage( DT_CUT ) <=
-            attack.dealt_dam.type_damage( DT_BASH ) ) ||
-        effects.count( "NO_EMBED" ) != 0 ) {
+
+    // We can only embed in monsters
+    bool embed = mon != nullptr && !mon->is_dead_state();
+    // And if we actually want to embed
+    embed = embed && effects.count( "NO_EMBED" ) == 0;
```

Comment:
Doesn't &= work for this?  I know it's supposed to be for bitwise operations, but I'm sure I've used it in the past for this, and it's worked.